shared/src/state.ts

```diff
@@ -90,6 +92,19 @@ export class ExerciseState {
     public readonly eocLog: readonly EocLogEntry[] = [];
     @IsString()
     public readonly participantId: string = '';
+
+    /**
+     * `Mutable<ExerciseState>` could still have immutable objects in spatialTree
+     */
```

Comment:
Using /** */ doesn't it highlight this info when dragging over spatialTrees?

Reply:
Yes, but we don't need to expose it like this. This comment is an implementation detail that could be useful when some weird bugs occur because you expect the state to be completely mutable you start debugging.